### PR TITLE
Cleaned up swift code to be more idiomatic.

### PR DESCRIPTION
**Pull request by toddsouthenbentley**

### CODE REVIEW

**Comment by pmconne** [comment]
@markschlosseratbentley mind reviewing?

File: test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-test-app/ViewController.swift [hunk 1]
```diff
@@ -10,103 +10,84 @@ import IModelJsNative
...
+        })
+        self.present(alert, animated: true)
+    }
+    
     @IBAction func onOpenSnapshotIModel(_ sender: Any) {
-        pickSnapshot();
+        pickSnapshot()
     }
+    
     func pickSnapshot(){
```
Comment (tcobbs-bentley) [inline test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-test-app/ViewController.swift:68]: While you're here, you should put a space before the open brace ({).

**Comment by tcobbs-bentley** [comment]
@pmconne, @markschlosseratbentley For what it's worth, I looked at these changes, and they look good to me, aside from one comment that I made.

**Review by pmconne — APPROVED** [review]
> I looked at these changes, and they look good to me

Thanks @tcobbs-bentley.

**Comment by aruniverse** [comment]
Can we update the code-owners, and add a rule for all *.swift files be reviewed by the mobile team?

File: test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-test-app/ViewController.swift (hunk 1 above)
Comment (toddsouthenbentley) [reply test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-test-app/ViewController.swift]: Fixed

**Comment by pmconne** [comment]
> Can we update the code-owners, and add a rule for all *.swift files be reviewed by the mobile team?

Might make more sense just to add the rule for everything in display-test-app/ios.
Shouldn't they also be code owners on /core/mobile? Currently that's owned by admins.

**Comment by toddsouthenbentley** [comment]
> > Can we update the code-owners, and add a rule for all *.swift files be reviewed by the mobile team?
> 
> Might make more sense just to add the rule for everything in display-test-app/ios. Shouldn't they also be code owners on /core/mobile? Currently that's owned by admins.

We haven't done much (any?) work on /core/mobile yet. Historically that's been done by others, but  I don't mind working on it.

**Review by markschlosseratbentley — COMMENTED** [review]
I see this merged in already, but just commenting that it also looks good to me.

**Pull request merged**